packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -61,6 +63,7 @@ const Transcribe = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': latestTranscribeRegions >= 1 ? 'ap-northeast-1' : 'ap-northeast-2',
   'ap-northeast-2': 'ap-northeast-2',
+  'ap-northeast-3': 'ap-northeast-1',
```

Comment:
Transcribe is not available in ap-northeast-3. Falling back to ap-northeast-1.

https://docs.aws.amazon.com/general/latest/gr/transcribe.html